packages/react-reconciler/src/ReactSideEffectTags.js

```diff
@@ -10,29 +10,29 @@
 export type SideEffectTag = number;
 
 // Don't change these two values. They're used by React Dev Tools.
-export const NoEffect = /*                 */ 0b00000000000000;
-export const PerformedWork = /*            */ 0b00000000000001;
+export const NoEffect = /*                     */ 0b000000000000000;
+export const PerformedWork = /*                */ 0b000000000000001;
 
 // You can change the rest (and add more).
-export const Placement = /*                */ 0b00000000000010;
-export const Update = /*                   */ 0b00000000000100;
-export const PlacementAndUpdate = /*       */ 0b00000000000110;
-export const Deletion = /*                 */ 0b00000000001000;
-export const ContentReset = /*             */ 0b00000000010000;
-export const Callback = /*                 */ 0b00000000100000;
-export const DidCapture = /*               */ 0b00000001000000;
-export const Ref = /*                      */ 0b00000010000000;
-export const Snapshot = /*                 */ 0b00000100000000;
-export const Passive = /*                  */ 0b00001000000000;
-export const PassiveUnmountPendingDev = /* */ 0b10000000000000;
-export const Hydrating = /*                */ 0b00010000000000;
-export const HydratingAndUpdate = /*       */ 0b00010000000100;
+export const Placement = /*                    */ 0b000000000000010;
+export const Update = /*                       */ 0b000000000000100;
+export const PlacementAndUpdate = /*           */ 0b000000000000110;
+export const Deletion = /*                     */ 0b000000000001000;
+export const ContentReset = /*                 */ 0b000000000010000;
+export const Callback = /*                     */ 0b000000000100000;
+export const DidCapture = /*                   */ 0b000000001000000;
+export const Ref = /*                          */ 0b000000010000000;
+export const Snapshot = /*                     */ 0b000000100000000;
+export const Passive = /*                      */ 0b000001000000000;
+export const Hydrating = /*                    */ 0b000010000000000;
+export const HydratingAndUpdate = /*           */ 0b000010000000100;
+export const Incomplete = /*                   */ 0b000100000000000;
```

Comment:
The reason these were in a separate section is that they aren't really effects, i.e. they don't have anything to  do with the commit phase. We just use this field to avoid adding another one.

Same with the new one you've added.